app/portal/account/page.js

```diff
@@ -2,22 +2,22 @@
 
 import styles from "./page.module.scss"
 import { SignOutBtn } from "@/components/auth/AuthButtons"
-import { useSession } from "next-auth/react";
+import { useSession } from "next-auth/react"
```

Comment:
By the way, this change was introduced by my implementation of `prettier` which we can go over some time in regards to code style consistency we'd like in the app.